green_cli/green.py

```diff
@@ -436,13 +437,24 @@ def getbalance(session, details):
 @green.command()
 @click.option('--subaccount', default=0, expose_value=False, callback=details_json)
 @click.option('--num-confs', default=0, expose_value=False, callback=details_json)
+@click.option('--all-coins', default=False, expose_value=False, callback=details_json)
 @with_login
 @print_result
 @gdk_resolve
 def getunspentoutputs(session, details):
     """Get unspent outputs"""
     return gdk.get_unspent_outputs(session.session_obj, json.dumps(details))
 
+@green.command()
+#@click.argument('--status', type=(UtxoUserStatus()), expose_value=False, multiple=True)
+@click.argument('status', type=(UtxoUserStatus()), expose_value=False, nargs=-1)
```

Comment:
Maybe it's worth adding a small help string here, otherwise the format of the status argument is not obvious without code inspection

```suggestion
@click.argument('status', type=(UtxoUserStatus()), expose_value=False, nargs=-1, help='Format is <txid>:<vout>:[default|frozen])
```

Reply:
Per https://click.palletsprojects.com/en/7.x/documentation/#documenting-arguments you can't document `click.argument`. I've added the format to the command help which gives:

````
$ green-cli setunspentoutputsstatus --help
Usage: green-cli setunspentoutputsstatus [OPTIONS] [STATUS]...

  Set unspent outputs status. Status format is
  <txid>:<vout>:[default|frozen]

Options:
  --help  Show this message and exit.
````